### DISCUSSION TITLE
Macro for navigate back in browser?

### DISCUSSION THREAD

**Discussion by nemanjam** [sezanzeb/input-remapper#793] [Q&A] [answered]
I have turned docs upside down and couldn't find simple macro for navigate back `Alt_Left + Arrow_Left` in browser. I am using Ubuntu.

I have tried this but it doesn't work:

```
modify(Alt_L, key(KEY_LEFT).w(300))
```

![image](https://github.com/sezanzeb/input-remapper/assets/9990165/db94e18e-3763-4d75-9b26-8c629c1e5597)

**Comment by jonasBoss** [comment, marked as answer]
you need to select the keyboard as a target in the dropdown on the left where you have currently the mouse selected. And `Alt_L + KEY_LEFT` should work.

If it does not work you might want to upgrade to the latest release 2.0.1

**Reply by nemanjam** [reply]
Now it works, thanks.
